backend/classes/Game.js

```diff
@@ -0,0 +1,21 @@
+
+class Game {
+  constructor(id) {
+    this.id = id;
+
+    this.playerList = {};
+
+    //Phase mean what is currently happen in the game - waiting for players, murderer Input? etc
+    this.phase = Game.PHASES.WAITING_FOR_PLAYERS
+  }
+
+  static PHASES ={
```

Comment:
enums should sit outside of the class

Reply:
Why? they are relevant to the class only

Reply:
because the enum is not a property of the class